modules/phase_field/src/auxkernels/TestEBSDAux.C

```diff
@@ -24,6 +24,11 @@ TestEBSDAux::TestEBSDAux(const InputParameters & parameters) :
 {
 }
 
+TestEBSDAux::~TestEBSDAux()
+{
+  delete _val;
```

Comment:
I'm merging this as-is but I want to change that interface to return a `MooseSharedPointer` instead.